java/server/src/org/openqa/selenium/events/zeromq/BoundZmqEventBus.java

```diff
@@ -24,7 +24,9 @@
 import org.zeromq.SocketType;
 import org.zeromq.ZContext;
 import org.zeromq.ZMQ;
+import zmq.io.mechanism.curve.Curve;
 
+import javax.swing.plaf.ColorUIResource;
```

Comment:
Looks like a mistaken import. Please remove.

Reply:
Thanks @shs96c !

Removed the unused import.

For the question: I assumed the same. @adamgoucher can shed some light?

Reply:
I don't think so. the 0mq bus is something we control 100% and this is just a secure default. (I expect most large implementations will use a different bus anyways)

Reply:
Thanks @adamgoucher